documentation/modules/ROOT/pages/04-quarkus-extensions.adoc

```diff
@@ -542,84 +547,40 @@ The REST call about should return an response like:
 }
 ----
 
+Then you can stop the container by typing kbd:[Ctrl+C].
+
 [#qext-add-probes-to-k8s]
 === Add probes to Kubernetes Quarkus Application
 
-Before we add the probes to the Kubernetes application, we need to push the rebuilt containers to the container registry.
+We need to push the rebuilt containers to the container registry.
 
 include::ROOT:partial$build-containers.adoc[tag=tag-push]
 
-Once the rebuilt containers has been successfully pushed to the container registry run the following command to add the probes to the kubernetes deployment.
+[#qext-health-check-k8s-delete-resources]
+=== Delete Kubernetes Resources
 
+include::ROOT:partial$k8s-build-deploy.adoc[tag=k8s-delete]
 
-ifndef::workshop[]
-[tabs]
-====
-kubectl::
-+
---
-[#qext-copy-k8s-probes-file]
-[source,bash,subs="+macros,+attributes"]
-----
-cp pass:[$TUTORIAL_HOME]/templates/probes.yaml pass:[$TUTORIAL_HOME]/work/{quarkus-project-name}/probes.yaml
-----
-copyToClipboard::qext-copy-k8s-probes-file[]
+[#qext-health-check-k8s-apply-resources]
+=== Apply Kubernetes Resources
 
-Apply the probes to the existing `pass:[$APP_NAME]` Kubernetes deployment:
+Since we are using Kubernetes extension, configured at xref:03-containers-kubernetes.adoc[Cloud Native Deployment] section, the kubernetes resource has been updated with both probes.
 
-[#qext-k8s-set-probes]
-[source,bash,subs="+macros,+attributes"]
-----
-kubectl patch deployment pass:[$APP_NAME] -p "$(cat pass:[$TUTORIAL_HOME]/work/{quarkus-project-name}/probes.yaml)"
-----
-copyToClipboard::qext-k8s-set-probes[]
---
-oc::
-+
---
-endif::[]
-[#qext-oc-set-probes]
+[#quarkusk8s-generated-kubernetes-resource-health-check]
 [source,bash,subs="+macros,+attributes"]
 ----
-oc set probe deployment pass:[$APP_NAME] --readiness=true --liveness=true --get-url=http://:8080/health
-----
-copyToClipboard::qext-oc-set-probes[]
-ifndef::workshop[]
---
-====
-endif::[]
-
-[#qext-invoke-k8s-svc]
-=== Invoke the Kubernetes Service
-
-Adding probes will cause the redeployment of the Kubernetes application. Wait for the application to be in `Running` state before you can xref:03-containers-kubernetes.adoc#conk8s-invoke-k8s-svc[Invoke the Service].  The status of the application can be checked with the command:
-ifndef::workshop[]
-[tabs]
-====
-kubectl::
-+
---
-[#qext-k8s-pod-status]
-[source,bash,subs="+macros,+attributes"]
+cat pass:[$TUTORIAL_HOME]/target/wiring-classes/META-INF/kubernetes/kubernetes.json
```

Comment:
@lordofthejars  can we use the yaml instead of JSON ?

Reply:
when it comes to workshop mode it will be `openshift.yaml` or `openshift.json`, we need have them set here with if and endif.

Reply:
Is there any difference in this case? I mean maybe you faced that users prefer YAML format.

About `openshift.yaml` is this something that is automatically set? I asked this because dekorate does not produce OpenShift files, but the `kubernetes.yaml` file totally valid in OpenShift, I have run it without any problem several times in the past.

Reply:
yeah,  YAML is easy to read. 

on OpenShift yaml am fine as long as we have instruction to say we expose svc

Reply:
Yes these instructions are in the `partial` defined so everywhere is updated.